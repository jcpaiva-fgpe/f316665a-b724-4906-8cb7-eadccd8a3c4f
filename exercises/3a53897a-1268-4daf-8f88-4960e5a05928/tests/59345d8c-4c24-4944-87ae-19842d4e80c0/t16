-1664 22222 -1656 22234
25
6 6
-6 -6
4 4
-8 -8
-18 18
10 -10
6 -6
4 -4
-2782 0
0 556
8000 0
1000 0
-8 8
8 -8
-10000 0
8 0
10000 0
2 2
100 -100
68 68
8200 0
0 -2000
-200 -200
50 0
-4 4
