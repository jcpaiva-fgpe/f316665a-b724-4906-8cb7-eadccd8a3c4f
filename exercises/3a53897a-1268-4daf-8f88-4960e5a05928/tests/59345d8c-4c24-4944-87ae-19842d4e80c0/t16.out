5
Atenas 1
Barcelona 3
Frankfurt 3
Madrid 5
Porto 4
